packages/desktop-client/src/components/accounts/Account.jsx

```diff
@@ -651,6 +658,23 @@ class AccountInternal extends PureComponent {
           this.setState({ showCleared: true });
         }
         break;
+      case 'toggle-reconciled':
+        const refilterTransactions = () => {
+          const filters = this.state.filters;
+          if (filters.length > 0) {
+            this.applyFilters([...filters]);
+          } else {
+            this.fetchTransactions();
```

Comment:
I think `fetchTransactions` accepts a `filters` parameter so you can just probably pass in the filters to that method and remove the if else block here. :)

Reply:
Updated